conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -58,8 +58,8 @@ def generate(self):
     content = t.load(MesonToolchain.cross_filename)
     assert "c_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7']" in content
     assert "c_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7']" in content
-    assert "cpp_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7']" in content
-    assert "cpp_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7']" in content
+    assert "cpp_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7', '-stdlib=libc++']" in content
+    assert "cpp_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7', '-stdlib=libc++']" in content
```

Comment:
It did not fail because of this:

```
# C/C++ arguments
c_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + preprocessor_definitions + deps_c_args
c_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + deps_c_link_args
cpp_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7', '-stdlib=libc++'] + preprocessor_definitions + deps_cpp_args
cpp_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7', '-stdlib=libc++'] + deps_cpp_link_args

# Objective-C/C++ arguments
objc_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + preprocessor_definitions + deps_c_args
objc_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + deps_c_link_args
objcpp_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + preprocessor_definitions + deps_cpp_args
objcpp_link_args = ['-isysroot', '/other/sdk/path', '-arch', 'myarch', '-otherminversion=10.7'] + deps_cpp_link_args
```
Objective-C/C++ arguments got the correct content 😓